clusterman/kubernetes/util.py

```diff
@@ -54,33 +54,36 @@
 
 
 class KubeApiClientWrapper:
+    _client = None
```

Comment:
might be worth a comment here re: why this is a classvar now (just to make sure that we can figure out why we did this even if someone moves this line around or something and the git blame no longer points directly at this commit) :)